src/Sentry/Extensibility/DisabledHub.cs

```diff
@@ -21,10 +21,10 @@ public class DisabledHub : IHub, IDisposable
 
         public void BindClient(ISentryClient client) { }
 
-        public Guid CaptureEvent(SentryEvent evt, Scope scope = null) => Guid.Empty;
+        public SentryId CaptureEvent(SentryEvent evt, Scope scope = null) => new SentryId(Guid.Empty);
 
```

Comment:
With the `implicit` operator in place, there shouldn't be a need to call the constructor here.
Can you simply return: `default` instead? Here and all other occurrences please.

Reply:
Even better would be to create a static readonly property called `Empty` inside `SentryId`
This way you can simply use: `SentryId.Empty`